### ISSUE TITLE
Increased plasma esterified carnitine

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5957]
**Preferred term label:** Increased plasma esterified carnitine

**Synonyms**

**Definition (free text, please give PubMed ID)** An increased concentration of esterified carnitine in the blood., PMID:6348429

**Parent term (use hpo.jax.org/app)** Abnormal circulating carnitine concentration HP:0010967

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
esterified carnitine is another way of saying acylcarnitine, I will add a synonym
